pkg/scheduler/api/v1/types.go

```diff
@@ -125,6 +127,11 @@ type LabelPreference struct {
 	Presence bool `json:"presence"`
 }
 
+// RequestedToCapacityRatioArguments holds arguments specific to RequestedToCapacityRatio priority function
+type RequestedToCapacityRatioArguments struct {
+	Shape string `json:"shape"`
```

Comment:
This is user visible. We need to explain in a comment how it is provided and what it means.
I would prefer the shape to be an array of (int, int) instead of a string that is parsed by the scheduler. Something like:

```
UtilizationShape []UtilizationShapePoint
```
```
type UtilizationShapePoint struct {
   Utilization int
   Score       int
}
```